### PR TITLE
Provider admin api with signing

### PR DESCRIPTION
**Pull request by HughParry**

### CODE REVIEW

File: packages/datasets/src/tests/mocks/data/captchas.json [hunk 1]
```diff
@@ -2,7 +2,7 @@
     "datasetId": "0x80f2e82ed079330c5fd9ea849a2b068e5b5c3473061219e5b33f2f210e6f1715",
     "captchas": [
         {
-            "salt": "0x01",
+            "salt": "0x0ee5e8130bf5834978971abbb3e94703",
```
Comment (forgetso) [inline packages/datasets/src/tests/mocks/data/captchas.json:5]: Why have these salts changed?

**Review by forgetso — CHANGES_REQUESTED** [review]
On track for world domination. Good work. One small Q. We can handle the commit stuff later when the populator is working.

File: packages/datasets/src/tests/mocks/data/captchas.json (hunk 1 above)
Comment (HughParry) [reply packages/datasets/src/tests/mocks/data/captchas.json:5]: IIRC, zod was getting angry about arg length for salts being under 34 chars

Comment (forgetso) [reply packages/datasets/src/tests/mocks/data/captchas.json:5]: Great

**Review by forgetso — APPROVED** [review]

**Pull request merged**